CODE/task-def.json

```diff
@@ -34,6 +34,10 @@
                 {
                     "valueFrom": "arn:aws:secretsmanager:eu-west-1:177420307256:secret:prod/penny/penny-bot/discord-bot-token-VmhBXH",
                     "name": "BOT_TOKEN"
+                },
+                {
+                    "valueFrom": "arn:aws:secretsmanager:eu-west-1:177420307256:secret:prod/penny/penny-bot/discord-bot-token-VmhBXH",
```

Comment:
I'm also not sure if we don't need to change this `valueFrom` key or not.

Reply:
Anyone having the link will be able to send logs to the channel just like us. So it's kind-of secret. The url doesn't have any other authorization method, other than being a long string that other people wouldn't be able to guess.

Reply:
I could just set it to send the logs directly to the channel with normal messages, too. I would prefer the webhook way, but it's not that important.

Reply:
Cool, it's no problem. If you set up the Webhook URL and DM me on Discord I can add it to secrets manager and update the ARN